tools/__init__.py

```
@@ -5,7 +5,7 @@
 # from .expert import GetExpertOpinionTool
 # from .playwright import WebNavigatorTool # Removed
 from .envsetup import ProjectSetupTool
-
+from .interpreter import InterpreterTool
```

Ruff (0.11.9)

`.interpreter.InterpreterTool` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)